code/game/gamemodes/meteor/meteors.dm

```diff
@@ -139,12 +139,20 @@ GLOBAL_LIST_INIT(meteorsC, list(/obj/effect/meteor/dust)) //for space dust event
 		get_hit()
 
 /obj/effect/meteor/proc/ram_turf(turf/T)
-	//first bust whatever is in the turf
+	//first bust whatever is in the turf or check if we hit a shield
 	for(var/atom/A in T)
 		if(A != src)
-			if(isliving(A))
-				A.visible_message("<span class='warning'>[src] slams into [A].</span>", "<span class='userdanger'>[src] slams into you!.</span>")
-			A.ex_act(hitpwr)
+			if(!istype(A,/obj/effect/ftl_shield))
+				if(isliving(A))
+					A.visible_message("<span class='warning'>[src] slams into [A].</span>", "<span class='userdanger'>[src] slams into you!.</span>")
+				A.ex_act(hitpwr)
+			else //We hit the ships shields
+				for(var/area/shuttle/ftl/area in world)
```

Comment:
I'm fairly sure we have a proc for this in SSstarmap or SSship. Plus this would send the audio to the FoB IIRC.